Projekty/Projekt1/Gryszkiewicz_Chylak/Wstępny_Model/model_wstepny.ipynb

```diff
@@ -0,0 +1,2038 @@
+{
```

Comment:
<p>Cenna uwaga. Należałoby jeszcze wspomnieć, że w tym przypadku zbiór był zbalansowany dlatego accuracy jest właściwą miarą. </p><p>Gdyby teoretycznie usunięto bardzo wiele wierszy na etapie inżynierii cech, balans mógłby ulec zmianie, dlatego też ważne jest aby wspomnieć jak wiele wierszy usunięto. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mini-pw/2021L-WUM/pull/147/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>